overrides/ios-override.json

```diff
@@ -32,6 +32,9 @@
         },
         "autofill": {
             "state": "enabled"
+        },
+        "customUserAgent": {
+            "state": "enabled"
```

Comment:
1. And @marcosholgado  will do the same for android during his project, right?

2. Is our android list of domains the same as iOS list? I'm guessing not, so should we move iOS domains into an overwrite or Marcos will do it?

3. to double-check -client implementation is not going into BSK and macOS is not affected, right?

Reply:
1. Yes

2. I haven't seen their list so we'll move it as needed in their project.

3. Correct not going in to BSK at this time.